Enable to run the generator

### Description

Running the generator reported an error.

### Steps to Reproduce

1. yo ovh-drupal-theme
2. answer questions

```txt
? Enter the new theme name: My Awesome Theme
? Enter the module machine name: my_awesome_theme
? Enter theme description: foo
? Enter package name: foo
? Enter Drupal Core version: 8.x
? Do you want to be a sub-theme? No
```

3. see error

**Expected behavior:**

Generator should work.

**Actual behavior:**

Here is the error reported:

```sh
npm ERR! code 1
npm ERR! Command failed: git checkout 4.0
npm ERR! error: pathspec '4.0' did not match any file(s) known to git
npm ERR!
```

**Reproduces how often:**

All the the time.

### Additional Information

Seems to be related to this error: ovh-ux/gulp-drupal-stack#22

--- a/generators/app/templates/_package.json
+++ b/generators/app/templates/_package.json
@@ -31,7 +31,7 @@
     "babel-loader": "^7.1.2",
     "babel-preset-es2015": "^6.24.1",
     "eslint-config-ovh": "^0.1.1",
-    "gulp": "github:gulpjs/gulp#4.0",
+    "gulp": "^4.0.0",
     "gulp-drupal-theme-core": "github:ovh-ux/gulp-drupal-theme-core",
     "rc": "^1.2.1",
     "stylelint-config-ovh": "^0.1.0",

